pkg/experiment/metadata.go

```diff
@@ -152,32 +194,34 @@ func (m *Metadata) connect() error {
 
 	m.session = session
 
-	if err := session.Query("CREATE KEYSPACE IF NOT EXISTS swan WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1};").Exec(); err != nil {
-		return err
+	if m.config.CassandraCreateKeyspace {
+		if err = m.createKeyspace(); err != nil {
+			return err
+		}
 	}
 
-	// NOTE: Schema consistency check is ignored by CREATE query. (https://git-wip-us.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=doc/native_protocol_v4.spec)
-	// To ensure schema consistency we perform a simple SELECT query on 'system_schema.keyspaces'.
-	// Consistency level is taken from 'cluster.Consistency' variable, it can also be defined for individual Query.
-	if err = session.Query("SELECT * FROM system_schema.keyspaces;").Exec(); err != nil {
-		return err
-	}
+	// // NOTE: Schema consistency check is ignored by CREATE query. (https://git-wip-us.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=doc/native_protocol_v4.spec)
```

Comment:
this comments could be dropped b/c I don't see any reason to check that keyspace is created